common/primitives/src/schema.rs

```diff
@@ -1,5 +1,19 @@
-use frame_support::BoundedVec;
+#[cfg(feature = "std")]
+use crate::utils;
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_std::prelude::*;
+#[cfg(feature = "std")]
+use utils::*;
 
 pub type SchemaId = u16;
 
-pub type Schema<T> = BoundedVec<u8, T>;
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Default, Clone, Encode, Decode, PartialEq, Debug, TypeInfo, Eq)]
+pub struct SchemaResponse {
+	pub schema_id: SchemaId,
+	#[cfg_attr(feature = "std", serde(with = "as_string"))]
```

Comment:
does this make sense or we want to encode `as_hex`?

Reply:
I think hex is a bit more fitting?

Reply:
we are not sure on this yet. Keeping it as string until it changes